spec/features/get_pacticipants_spec.rb

```diff
@@ -0,0 +1,35 @@
+describe "Get pacticipants" do
+
+  let(:path) { "/pacticipants" }
+  let(:response_body_hash) { JSON.parse(subject.body, symbolize_names: true) }
+  let(:expected_response_body) { {name: "Foo"} }
+
+  subject { get path; last_response }
```

Comment:
You've copied from one of my very old specs. You don't need the `; last_response`. I'm slowly removing all of them. You just need the `get(path)`.